README.md

```diff
@@ -117,6 +117,43 @@ This is an example response body:
 }
 ```
 
+### POST - Adding draft editorial annotations and writing them in PAC
+
+Using curl:
+
+```
+curl http://localhost:8080/draft/content/{content-uuid}/annotations -X POST --data '{
+        {
```

Comment:
This "{" and its closing partner are extra.

Also, I would suggest us using a real example body data, so that the user knows what is the expected format for the "id" and "predicate" fields.